Source/Core/DolphinQt2/GameList/GameFile.cpp

```diff
@@ -208,6 +206,21 @@ void GameFile::SaveCache()
   // TODO
 }
 
+QString GameFile::GetFileName() const
+{
+  return QFileInfo(m_path).fileName();
```

Comment:
I see it more akin to "committing a compile error that's a duplicate of another compile error": fixing the original problem, while it also fixes the duplicate problem, can stand on its own.